prosody/Dockerfile

```diff
@@ -1,11 +1,24 @@
 FROM jitsi/base
 
+ADD https://raw.githubusercontent.com/jitsi/jitsi-meet/05b7e6facc25422fba6a4c264e8f42411910019d/resources/prosody-plugins/token/util.lib.lua /prosody-plugins/token/util.lib.lua
+ADD https://raw.githubusercontent.com/jitsi/jitsi-meet/05b7e6facc25422fba6a4c264e8f42411910019d/resources/prosody-plugins/mod_token_verification.lua /prosody-plugins/mod_token_verification.lua
+ADD https://raw.githubusercontent.com/jitsi/jitsi-meet/05b7e6facc25422fba6a4c264e8f42411910019d/resources/prosody-plugins/mod_auth_token.lua /prosody-plugins/mod_auth_token.lua
+
 RUN \
 	apt-dpkg-wrap apt-get update && \
+	apt-dpkg-wrap apt-get install -y lua5.2 liblua5.2-dev libssl1.0-dev luarocks gcc git && \
 	apt-dpkg-wrap apt-get install -t stretch-backports -y prosody && \
-	apt-cleanup && \
 	rm -rf /etc/prosody
 
+RUN \
+  luarocks install lua-cjson 2.1.0-1 && \
+	luarocks install luajwtjitsi && \
+	luarocks install basexx
```

Comment:
This one we have on Debian: https://packages.debian.org/stretch/lua-basexx

Reply:
I wanted to use the same methods like `jitsi-meet-tokens`.
https://github.com/jitsi/jitsi-meet/blob/master/debian/jitsi-meet-tokens.postinst#L68

Why do we want to do it differently?

Reply:
If it works as expected I'll also make the change in the jitsi-meet-tokens package.

Reply:
Done.